app/services/duplicate_application.rb

```diff
@@ -20,6 +20,8 @@ def duplicate
     )
 
     ApplicationForm.create!(attrs).tap do |new_application_form|
+      update_work_history_status_if_nil(new_application_form)
+
```

Comment:
Instead of this separate method, I'd just change the code above to:
```ruby
).merge(
  phase: target_phase,
  previous_application_form_id: original_application_form.id,
  recruitment_cycle_year: @recruitment_cycle_year,
  work_history_status: original_application_form.work_history_status || 'can_complete',
)
```